collectors/pod.go

```diff
@@ -137,6 +137,18 @@ var (
 		"The limit on memory to be used by a container in bytes.",
 		[]string{"namespace", "pod", "container", "node"}, nil,
 	)
+
+	descPodContainerResourceRequestsGPU = prometheus.NewDesc(
+		"kube_pod_container_resource_requests_gpu",
```

Comment:
Actually I made a mistake here, the limit can be changed in all directions so the `_total` suffix is actually not appropriate to be used here.